public/app/themes/justice/src/sass/editor.scss

```diff
@@ -1,2 +1,7 @@
 @import "../components/search/editor";
 @import "../components/simple-definition-list-blocks/editor";
+
+// Triple the height of the suggestion list for the combobox component
+.components-form-token-field__suggestions-list {
+  max-height: 384px !important;
+}
```

Comment:
With an aim for consistency, could you add a line break at the end of the file.

Reply:
Thanks @EarthlingDavey that should be fixed :)